.github/workflows/kubernetes.yml

```diff
@@ -53,15 +53,15 @@ jobs:
           kind create cluster
       - name: Build and Load Docker images
         run: |
-          cd docker
-          cp ../target/release/{linera,linera-proxy,linera-server} .
+          cd docker/local-tests
```

Comment:
Moved these to this `local-tests` folder so it's clear for people what the Dockerfile in there does, and it doesn't get confused with the "prod" one in the repo root